try:
	print(1/0)
except Exception as p:
	print(p)